### ISSUE TITLE
Not reactive?

### ISSUE THREAD

**Issue by Reggino** [AllenFang/react-bootstrap-table#2]
Hi,

This is cool stuff, thanks. However, i'm having issues getting updated data in the table. I'm having something like:

```
...
render () {
  return <BootstrapTable data={ this.state.data }>
	...
 </BootstrapTable>
...
```

After the first render, ```this.state.data``` is updated. However, the updated data is not rendered in the table.

**Comment by Reggino** [comment]
Maybe one should be able to pass in a ```TableDataStore``` that could extend something like ```https://github.com/christianalfoni/flux-react-store ``` ? That way the store could emit a "change" event or something like that to detect the change of the original data.

**Comment by AllenFang** [comment]
Hi @Reggino,

It's a good question for react-bootstrap-table, someone asked that before, maybe it just like "update data on the fly".

And I will spend some times to review your PR, anyway your idea is very great, I like it :)

Thanks for your contribution.

**Comment by yorzi** [comment]
Hi @AllenFang thanks for making this cool component, I encountered the same issue that @Reggino described above. Will you review his PR and merge the changes recently?

**Comment by AllenFang** [comment]
Hi @yorzi, 
I have created a new branch <a href="https://github.com/AllenFang/react-bootstrap-table/tree/feature03">feature03</a> based on @Reggino 's PR and did a little change.

Currently, it's worked well on <a href="https://github.com/AllenFang/react-bootstrap-table/tree/feature03">feature03</a> .

Anyway, I think I will merge back to master and release before this friday.

Sorry, let you wait it so long :) if you have anything that you want, welcome to let me know:)

**Comment by yorzi** [comment]
@AllenFang :+1: thank you!

**Comment by AllenFang** [comment]
Hi @yorzi,
I've released v0.8.0 for this issue, you can visit <a href='http://allenfang.github.io/react-bootstrap-table/example.html#fly'>here</a> and <a href='http://allenfang.github.io/react-bootstrap-table/docs.html#about-tabledataset'>here</a> for more detail

**Comment by yorzi** [comment]
@AllenFang cool, thanks!
